requirements.txt

```
@@ -1,7 +1,7 @@
 black==22.3.0
 flake8==5.0.4
 flake8-tidy-imports==4.6.0
-graviton@git+https://github.com/algorand/graviton@26d47e31c3ffb5132cd29251eefcb50db0723d03
+graviton@git+https://github.com/tzaffi/graviton@exec-multi-dispatch
```

```suggestion
graviton@git+https://github.com/tzaffi/graviton@v0.8.0
```

TODO: make this change after creating graviton tag `v0.8.0` and before merging.